website_variants_extra/static/src/js/test_comments_qty.js

```diff
@@ -0,0 +1,43 @@
+(function(){
+    'use strict';
+    openerp.Tour.register({
+        id: 'shop_test_comment_qty',
+        name: 'Test Product Name with Variants',
+        path: '/shop/product/ipad-retina-display-5',
+        mode: 'test',
+        steps: [
+            {
+                title:     "Write first comment",
+                onload:    function(Tour){
+                               $('#comment textarea').text('1st Comment');
+                               $('#comment .fa-star-o:last').trigger('click');
+                            },
+                element:  'form a:contains(Post)',
+            },
+            {
+                title:     "Publish a comment",
+                onload:    function(Tour){
+                                $('#comments-list .btn-danger').first().trigger('click');
+                           },
+                waitFor:   '#comments-list:first:contains(1st)',
+                element:   '#comments-list',
+            },
+            {
+                title:     "Post second comment",
+                onload:    function(Tour){
+                               $('#comment textarea').text('2nd Comment');
+                               $('#comment .fa-star-o:last').trigger('click');
+                            },
+                waitFor:   '#comments-list:first:contains(1st)',
+                element:  'form a:contains(Post)',
+            },
+            {
+                title: 'check comment qty',
+                waitFor: '#no-reviews:contains(1)',
```

Comment:
@nhomar,  the quantity it's verified, 
the test behavior it's the following:
1. Create a Comment
2. Publish the first comment
3. Create another comment
4. Verify that only published comment was taken in acconunt.
Best regards.